src/Handlers/TaxonomiesHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace GravApi\Handlers;
+
+use GravApi\Resources\PageCollectionResource;
+
+/**
+ * Class TaxonomiesHandler
+ * @package GravApi\Handlers
+ */
+class TaxonomiesHandler extends BaseHandler
+{
+    public function getTaxonomies($request, $response, $args)
+    {
+        $parsedBody = $request->getParsedBody();
+        $operation = $parsedBody['operation'] ?? 'or';
+        $filter = $parsedBody['taxonomyFilter'] ?? array();
+        $collection = $this->grav['taxonomy']->findTaxonomy($filter, strtolower($operation));
+
+        $resource = new PageCollectionResource($collection);
```

Comment:
I agree to move it to a POST /api/pages/searches endpoint (the POST and searches according to my comment below)

Reply:
Yes, the config setup is perhaps not flexible enough at the moment. I hadn't previously considered nested endpoints like this. 

Since the `/searches` endpoint is simply returning a collection of Pages, perhaps for now we just use the Pages GET config and I'll make an issue to address this problem going forward? I think it makes sense that if you can get _all_ pages, then it's probably fine to inherit the same settings and allow users to be able to search for a subset of those pages.

Reply:
Okay, for now that is ok to me. It might be a little bit confusing for someone when doing configuration so I'll see whether I can document this in order to make that clear. I'll also add the tests (shame on me for forgetting about that).

Reply:
Cool, let's proceed with it that way for now. :+1:

I've made a new issue #57 to address the config situation.